### PR TITLE
Add procedure to deploy langflow on AWS 

### PR DESCRIPTION
**Pull request by kazuki306**
Related to #1052 

Created procedure to deploy langflow on AWS with AWS CDK.

### CODE REVIEW

File: src/backend/langflow/services/settings/base.py
Comment (ogabrielluiz) [inline src/backend/langflow/services/settings/base.py:1]: I think we should roll back the changes in langflow itself and everything should be good to go.

File: src/backend/langflow/settings.py
Comment (ogabrielluiz) [inline src/backend/langflow/settings.py:1]: This should be reverted too.

File: src/backend/langflow/alembic.ini
Comment (ogabrielluiz) [inline src/backend/langflow/alembic.ini:1]: By not defining it, will it still work if I just run the alembic CLI? Is it possible to define the URL when calling auto revision or upgrade?

**Review by ogabrielluiz — CHANGES_REQUESTED** [review]

**Comment by satyagofylo** [comment]
hi @kazuki306 thanks for sharing the deployment 

i tried to deploy the same using the tutorial, however i was not successful 

once i tried **cdk deploy** command 

`cdk deploy


✨  Synthesis time: 17.93s

current credentials could not be used to assume 'arn:aws:iam::692536198005:role/cdk-hnb659fds-deploy-role-692536198005-us-west-2', but are for the right account. Proceeding anyway.
current credentials could not be used to assume 'arn:aws:iam::692536198005:role/cdk-hnb659fds-deploy-role-692536198005-us-west-2', but are for the right account. Proceeding anyway.
current credentials could not be used to assume 'arn:aws:iam::692536198005:role/cdk-hnb659fds-deploy-role-692536198005-us-west-2', but are for the right account. Proceeding anyway.
current credentials could not be used to assume 'arn:aws:iam::692536198005:role/cdk-hnb659fds-deploy-role-692536198005-us-west-2', but are for the right account. Proceeding anyway.
current credentials could not be used to assume 'arn:aws:iam::692536198005:role/cdk-hnb659fds-deploy-role-692536198005-us-west-2', but are for the right account. Proceeding anyway.
LangflowAppStack:  start: Building e77031893275c08bcaa0a774aa8b611727afd045b3b5d8e1e6f0f04063d9d386:current_account-current_region
LangflowAppStack:  success: Built e77031893275c08bcaa0a774aa8b611727afd045b3b5d8e1e6f0f04063d9d386:current_account-current_region
LangflowAppStack:  start: Building 2787cf9e14cb283f5bd620e092b3693febd0ead397441b7fe0deffdf2b93ddd0:current_account-current_region
LangflowAppStack:  success: Built 2787cf9e14cb283f5bd620e092b3693febd0ead397441b7fe0deffdf2b93ddd0:current_account-current_region
LangflowAppStack:  start: Publishing e77031893275c08bcaa0a774aa8b611727afd045b3b5d8e1e6f0f04063d9d386:current_account-current_region
LangflowAppStack:  start: Building 94b83655403dac8fe335f8604b61c01d32209e0547652d8a1026bc2049365427:current_account-current_region
LangflowAppStack:  success: Built 94b83655403dac8fe335f8604b61c01d32209e0547652d8a1026bc2049365427:current_account-current_region
LangflowAppStack:  start: Publishing 2787cf9e14cb283f5bd620e092b3693febd0ead397441b7fe0deffdf2b93ddd0:current_account-current_region
LangflowAppStack:  start: Building 91dded8d95bb1c65225d536db769102658af3236db20060f8104ba7fff8957c0:current_account-current_region
LangflowAppStack:  start: Publishing 94b83655403dac8fe335f8604b61c01d32209e0547652d8a1026bc2049365427:current_account-current_region
current credentials could not be used to assume 'arn:aws:iam::692536198005:role/cdk-hnb659fds-file-publishing-role-692536198005-us-west-2', but are for the right account. Proceeding anyway.
current credentials could not be used to assume 'arn:aws:iam::692536198005:role/cdk-hnb659fds-file-publishing-role-692536198005-us-west-2', but are for the right account. Proceeding anyway.
current credentials could not be used to assume 'arn:aws:iam::692536198005:role/cdk-hnb659fds-file-publishing-role-692536198005-us-west-2', but are for the right account. Proceeding anyway.
LangflowAppStack:  success: Published e77031893275c08bcaa0a774aa8b611727afd045b3b5d8e1e6f0f04063d9d386:current_account-current_region
LangflowAppStack:  success: Published 94b83655403dac8fe335f8604b61c01d32209e0547652d8a1026bc2049365427:current_account-current_region
ERROR: Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?
LangflowAppStack:  fail: docker build --build-arg BACKEND_URL=http://backend.ecs-deploy.com:7860 --tag cdkasset-91dded8d95bb1c65225d536db769102658af3236db20060f8104ba7fff8957c0 --file cdk.Dockerfile --platform linux/amd64 . exited with error code 1: ERROR: Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?
LangflowAppStack:  success: Published 2787cf9e14cb283f5bd620e092b3693febd0ead397441b7fe0deffdf2b93ddd0:current_account-current_region

 ❌ Deployment failed: Error: Failed to publish asset 2787cf9e14cb283f5bd620e092b3693febd0ead397441b7fe0deffdf2b93ddd0:current_account-current_region
    at Deployments.publishSingleAsset (/usr/local/lib/node_modules/aws-cdk/lib/index.js:470:11458)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async Object.publishAsset (/usr/local/lib/node_modules/aws-cdk/lib/index.js:470:177912)
    at async /usr/local/lib/node_modules/aws-cdk/lib/index.js:470:163339

Failed to publish asset 2787cf9e14cb283f5bd620e092b3693febd0ead397441b7fe0deffdf2b93ddd0:current_account-current_region`

also when i tried to reuse the command again i got no space limit error

Can you please guide me and tell if i am doing any thing wrong in the process

**Pull request merged**